lib/tasks/aws.rake

```diff
@@ -9,8 +9,13 @@ namespace :atc do
       checksum_algorithm_name = ENV['checksum_algorithm_name']
 
       job_identifier = "fixity-check-from-rake-task-#{SecureRandom.uuid}"
-      remote_fixity_check = Atc::Aws::RemoteFixityCheck.new(CHECK_PLEASE['ws_url'], CHECK_PLEASE['auth_token'])
-      response = remote_fixity_check.perform(job_identifier, bucket_name, object_path, checksum_algorithm_name)
+      remote_fixity_check = Atc::Aws::RemoteFixityCheck.new(
+        CHECK_PLEASE['http_base_url'], CHECK_PLEASE['ws_url'], CHECK_PLEASE['auth_token']
+      )
+      response = remote_fixity_check.perform(
```

Comment:
Update to use HTTP instead of WEBSOCKET?

Reply:
Good suggestion!  I kept WEBSOCKET initially because I was thinking of WEBSOCKET as the "primary" method of communication, but treating HTTP as our "primary" method for now (given server constraint) is a good way to go.  I just pushed that change.